.github/dependabot.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+updates:
+- package-ecosystem: github-actions
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: daily
+  open-pull-requests-limit: 10
+  reviewers:
+  - alexanderbez
+  - fedekunze
```

Comment:
not nessecary to add more people here as codeowners will fire and invite the usual people